fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/mixin/object/builder/AbstractBlockSettingsAccessor.java

```diff
@@ -27,12 +27,73 @@
 import net.minecraft.block.Block;
 import net.minecraft.block.BlockState;
 import net.minecraft.block.MapColor;
+import net.minecraft.block.Material;
 import net.minecraft.sound.BlockSoundGroup;
 import net.minecraft.util.Identifier;
 
 @Mixin(AbstractBlock.Settings.class)
 public interface AbstractBlockSettingsAccessor {
+	/* GETTERS */
+	@Deprecated
+	@Accessor
+	Material getMaterial();
+
+	@Deprecated
+	@Accessor
+	float getHardness();
+
+	@Deprecated
+	@Accessor
+	float getResistance();
+
+	@Deprecated
+	@Accessor
+	boolean getCollidable();
+
+	@Deprecated
+	@Accessor
+	boolean getRandomTicks();
+
+	@Deprecated
+	@Accessor("luminance")
+	ToIntFunction<BlockState> getLuminance();
+
+	@Deprecated
+	@Accessor
+	Function<BlockState, MapColor> getMapColorProvider();
+
+	@Deprecated
+	@Accessor
+	BlockSoundGroup getSoundGroup();
+
+	@Deprecated
+	@Accessor
+	float getSlipperiness();
+
+	@Deprecated
+	@Accessor
+	float getVelocityMultiplier();
+
+	@Deprecated
+	@Accessor
+	boolean getDynamicBounds();
+
+	@Deprecated
+	@Accessor
+	boolean getOpaque();
+
+	@Deprecated
+	@Accessor
+	boolean getIsAir();
+
+	@Deprecated
+	@Accessor
+	boolean isToolRequired();
```

Comment:
A tiny nitpick here:
```suggestion

	boolean isToolRequired();
```

Reply:
"Suggestion cannot be identical to original text"

Reply:
huh, I guess it thought the new line was an unnecessary whitespace for some reason; Welp, I guess it'll have to be done manually then